### ISSUE TITLE
5.8.0 Infill after change pattern doesn't work Using the standard Fast - Engeneering -0.2mm profile

### ISSUE THREAD

**Issue by marcovalstar** [Ultimaker/Cura#19640]
### Cura Version

5.8.0

### Operating System

Windows 11

### Printer

Ulitmaker S5

### Reproduction steps

I make a new project and choose my printing profile
Than i load my models (2 models that are merged because of printing 2 different colors)
Then i sliced the model.
Not satisfied from the result i change the infill dencity and/or the pattern but that doesn't have any effect after re-slicing

### Actual results

Using the standard Fast - Engeneering -0.2mm profile
The re-slicing is the same as the result before the changes

### Expected results

I expect the infill changed from for example 25% to 75% and the standard pattern (grid) changes to the choosen (Gyroid)

### Add your .zip and screenshots here ⬇️

[Change_infill.zip](https://github.com/user-attachments/files/16959551/Change_infill.zip)

**Comment by GregValiant** [comment]
Thanks for the report.
We ask for project files so we get your Cura settings, your printer, and the model.  Without it I would be guessing and I respectfully refuse to do that.

**Comment by marcovalstar** [comment]
[Error_slicing_VRT.zip](https://github.com/user-attachments/files/16964112/Error_slicing_VRT.zip)

Sorry for not given all the information. Here it is.

**Comment by GregValiant** [comment]
There is so little infill in the letters, and the infill is just a single layer.  It's hard to tell what is going on.
This is my slice of the project with no changes (E2 is Gyroid at 25% and E1 is Triangles at 20%)
![image](https://github.com/user-attachments/assets/0ec6b1cc-8ceb-4a30-bc3d-a1ac28cffaae)
When I change E2 to 75%:
![image](https://github.com/user-attachments/assets/807ed5f5-d39a-4591-b6af-3248b02783af)

When I scale the model up it slices like this which appears is correct.  This is with E2 left at your 25% infill density.
![image](https://github.com/user-attachments/assets/48aba69c-a04f-4df0-979d-c32070c58df5)

I don't see a bug here and I can't duplicate the problem.  Your project does not contain the settings you describe above and the CuraProfile you provided does not contain the settings you describe above.
When I change the infill density, the infill density changes.  The infill areas of the letters are so small and shallow that they would not require any infill at all.

I'm going to remove the bug from this.  I'm inclined to close it as well.  Let me know if you feel that is premature.

**Comment by GregValiant** [comment]
I can't duplicate this.  I change the extruder settings and they are reflected in the slice.  I'm using your project, I just scaled it up a bit so the infill areas are larger.
I have changed the infill pattern and density a bunch of different ways and it always appears in the print per my settings.
I'll put the bug label back on this for you and someone from the Cura team ( @HellAholic ??) will take a look.  

This happens to be with "Lines" and a custom sine wave infill I designed.  Everything worked as expected.
![image](https://github.com/user-attachments/assets/bacb51b3-86f6-4b04-bbbc-aa413e19f419)

Here I've switched patterns and density and tile size from one part to the other.
![image](https://github.com/user-attachments/assets/ac714814-eb96-4ba7-8609-ad0e8d6266c0)
